Dockerfile.arm

```diff
@@ -0,0 +1,207 @@
+# This file describes the standard way to build Docker, using docker
+#
+# Usage:
+#
+# # Assemble the full dev environment. This is slow the first time.
+# docker build -t docker .
+#
+# # Mount your source in an interactive container for quick testing:
+# docker run -v `pwd`:/go/src/github.com/docker/docker --privileged -i -t docker bash
+#
+# # Run the test suite:
+# docker run --privileged docker hack/make.sh test
+#
+# # Publish a release:
+# docker run --privileged \
+#  -e AWS_S3_BUCKET=baz \
+#  -e AWS_ACCESS_KEY=foo \
+#  -e AWS_SECRET_KEY=bar \
+#  -e GPG_PASSPHRASE=gloubiboulga \
+#  docker hack/release.sh
+#
+# Note: AppArmor used to mess with privileged mode, but this is no longer
+# the case. Therefore, you don't have to disable it anymore.
+#
+
+FROM ioft/armhf-ubuntu:14.04
+MAINTAINER Govinda Fichtner <govinda.fichtner@googlemail.com> (@_beagile_)
+
+RUN	apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys E871F18B51E0147C77796AC81196BA81F6B0FC61
```

Comment:
Seems to be a network problem. This line is identical to the official `Dockerfile`.
I just tried the following on a Raspberry Pi 2 with HypriotOS installed:

```
apt-get update && apt-get install make
git clone https://github.com/hypriot/docker
cd docker
git checkout 17802-add-initial-support-for-docker-on-arm
make binary
```

And it works:

```
Step 2 : RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys E871F18B51E0147C77796AC81196BA81F6B0FC61
 ---> Running in 0005ec008883
Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.UtLZNZn9Rw --no-auto-check-trustdb --trust-model always --keyring /etc/apt/trusted.gpg --primary-keyring /etc/apt/trusted.gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys E871F18B51E0147C77796AC81196BA81F6B0FC61
gpg: requesting key F6B0FC61 from hkp server p80.pool.sks-keyservers.net
gpg: key F6B0FC61: public key "Launchpad PPA for Native ZFS for Linux" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
 ---> 799a2b5a993d
Removing intermediate container 0005ec008883
```